README.md

```diff
@@ -1,7 +1,77 @@
 # Grateful Together API
 
+## Abstract: 
+This is the Backend API for the [Grateful Together App](https://www.FIX-THIS.com). Grateful Together is an application that allows groups of people to keep a central, easy to use, daily gratitude journal.
 
-### JSON Contract
+### Preview of App:
+Live Deployment link
+[Frontend Repo](https://github.com/GratefulTogether/gratefultogether-ui) 
+
+Add GIF here
+
+### Context:
```

Comment:
I would change "Context" to be something more like "Summary" of "Description" and then a 1-2 sentence blurb about what the app does. Maybe one sentence about the overall app (ex: "This app allows users to journal about their day") and another about this specific app (ex: "This application exposes the endpoints necessary for logging user journal entries").